tilelang/metal/language/__init__.py

```
@@ -0,0 +1,25 @@
+from __future__ import annotations
+
+import importlib
+
+from tilelang.language.common import *  # noqa: F401,F403
+from tilelang.language.common import __all__ as _COMMON_ALL
+
+from .tir import *  # noqa: F401,F403
+from .tir import __all__ as _TIR_ALL
+
+__tilelang_dialect__ = "metal"
+__all__ = tuple((*_COMMON_ALL, "MPSIntrinEmitter", *_TIR_ALL))
```

Ruff (0.15.21)

[error] Invalid object in `__all__`, must contain only strings

(PLE0604)